mod_reforged/hooks/entity/tactical/humans/noble_arbalester.nut

```diff
@@ -11,8 +11,8 @@
 		this.setAppearance();
 		this.getSprite("socket").setBrush("bust_base_military");
 		this.m.Skills.add(::new("scripts/skills/perks/perk_bullseye"));
+		this.m.Skills.add(::new("scripts/skills/perks/perk_rf_entrenched"));
 		this.m.Skills.add(::new("scripts/skills/perks/perk_rotation"));
-		this.m.Skills.add(::new("scripts/skills/perks/perk_mastery_crossbow"));
```

Comment:
I'd keep the Crossbow Mastery on these guys just for flavor reasons even if it does nothing mechanically for them. These guys are noble crossbowmen and them not having this mastery seems bad. This is one of those cases where I think flavor is super important compared to the gain of fewer perks to see on the tooltip.

Reply:
They're getting it through onSetupEntity so it's redundant here